P/power_grid_model_c/build_tarballs.jl

```diff
@@ -0,0 +1,59 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "power_grid_model_c"
+version = v"1.12.0"
+
+# Collection of sources required to complete build
+sources = [
+           ArchiveSource("https://github.com/PowerGridModel/power-grid-model/releases/download/v$(version)/power_grid_model-$(version).tar.gz", "b38be158af11541759b7b2c01e1baab099f8f22fe453237d7814a8698bb67745"),
+           DirectorySource("./bundled")
+          ]
+
+# Bash recipe for building across all platforms.
+script = raw"""
+apk del cmake
+cd $WORKSPACE/srcdir
+for f in ${WORKSPACE}/srcdir/patches/cmake-patch.patch; do
+    atomic_patch -p1 ${f}
+done
+cd power_grid_model-1.12.0/power_grid_model_c
+cmake -B build -DCMAKE_INSTALL_PREFIX=${prefix} -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} -DCMAKE_CXX_STANDARD=20 -DCMAKE_CXX_FLAGS="$CXXFLAGS" -DCMAKE_BUILD_TYPE=Release
+cmake --build build --parallel ${nproc}
+cmake --install build
+install_license $WORKSPACE/srcdir/power_grid_model-1.12.0/LICENSE
+"""
+
+platforms = [
+             Platform("x86_64", "linux"; libc = "glibc", cpu_target="x86_64_v2", cxxstring_abi=:cxx11),
+             Platform("x86_64", "linux"; libc = "glibc", cpu_target="x86_64_v3", cxxstring_abi=:cxx11),
```

Comment:
What are you trying to accomplish with all these `cpu_target` flags added to the platform? That flag is not provided by BinaryBuilder, and it doesn't appear you use them in the above build recipe at all, so how it supposed to be used?